lddtree.sh

```diff
@@ -314,9 +314,9 @@ shift $(( $OPTIND - 1))
 
 ${SET_X} && set -x
 
-if which scanelf >/dev/null; then
+if which scanelf &>/dev/null; then
```

Comment:
i think the `&>/dev/null` is bashism and not POSIX shell. We should probably do `>/dev/null 2>&1`